shared/src/api/extension/flatExtensionApi.ts

```diff
@@ -33,7 +35,7 @@ export type PartialWorkspaceNamespace = Omit<
 >
 /**
  * Holds internally ExtState and manages communication with the Client
- * Returns initialized public Ext API ready for consumption and API object marshaled into Client
+ * Returns initialized public Ext API pieces ready for consumption and internal Ext API ready to be passed to the main thread
```

Comment:
Thanks, this is easier to understand 🙌 

Minor suggestion to distinguish between "extension API" and "extension host API":
```suggestion
 * Returns the initialized public extension API pieces ready for consumption and the internal extension host API ready to be exposed to the main thread
```